dev/tools/svg2dart/bin/main.dart

```diff
@@ -0,0 +1,89 @@
+import 'dart:io';
+
+import 'package:args/args.dart';
+import 'package:svg2dart/svg2dart.dart';
+
+const String kCodegenComment =
+  '// AUTO GENERATED FILE DO NOT EDIT!\n'
+  '// This file was generated by svg2dart.\n';
+
+void main(List<String> args) {
+  final ArgParser parser = new ArgParser();
+
+  parser.addFlag(
+      'help',
+      abbr: 'h',
+      negatable: false,
+      help: 'Display the tool\'s usage instructions and quit.');
+
+  parser.addOption(
+      'output',
+      abbr: 'o',
+      help: 'Target path to write the generated Dart file to.');
+
+  parser.addOption(
+      'asset_name',
```

Comment:
for argument names we probably prefer hyphens?

Reply:
done